src/treasuryDAO_storage.mligo

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+type contract_extra =
+  { frozen_scale_value : nat option
+  ; frozen_extra_value : nat option
+  ; max_proposal_size : nat option
+  ; slash_scale_value : nat option
+  ; slash_division_value : nat option
+  ; min_xtz_amount : tez option
+  ; max_xtz_amount : tez option
```

Comment:
:thinking: I am confused: why are these `option`s?

If they are required in every case (as I think they are) then shouldn't they always have a defined value?

I took this as an example, but the same is the case for the others as well.

Reply:
Good catch. I think I made a blunder here.